uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/token/TokenMvcMockTests.java

```diff
@@ -858,7 +858,7 @@ void token_endpoint_should_return_Basic_WWW_Authenticate_Header() throws Excepti
                 .andExpect(
                         header()
                                 .stringValues("WWW-Authenticate",
-                                        "Basic realm=\"UAA/client\", error=\"unauthorized\", error_description=\"Bad credentials\"")
+                                        "Basic realm=\"UAA/client\"")
```

Comment:
Is this meant to remove? Instead of replace or adjust the error description?

Reply:
solved